### ISSUE TITLE
ParseUser Still broken

### ISSUE THREAD

**Issue by PauloxD** [thiagolocatelli/parse4j#35]
I can't seem to be able to even log in. If I use the login() method, I get an error

 "Exception in thread "pool-1-thread-7" java.lang.IllegalArgumentException: ParseFile must be saved before being set on a ParseObject."

If I use the logIn() method, I can only return the username. All others return null.

If I use the loginInBackground() Method, I receive the error:

"Inherited abstract methods are not accessible and could not be implemented"

Is anyone getting these errors or know how to fix?

Thanks

**Comment by nickolayrusev** [comment]
Yes, there is a problem with endpoint name. The bug is introduced from the commit with hash 
a58eae9 . May be the committer didn't run the test cases. May be this commit should be reverted.

**Comment by PauloxD** [comment]
Is there anything we can do in the meantime? It doesn't seem to be updated regularly.

**Comment by nickolayrusev** [comment]
The repo is updated regularly. If you create pull request the maintainer will review it and will merge into the next release. without new build everything that I will suggest to do will be a very dirty hack. 
If you want dirty hack -  see the commit a58eae9 and just copy the content of the ParseUser class into new class - MyParseUser and replace "_User" with "users" (revert it manually) and wherever you want to use ParseUser you will use MyParse user.

**Comment by passa763** [comment]
I did a little hack with this where when a user logs in with .login() it doesn't do a check on the ParseFiles. See my commit [f4c024e]
(https://github.com/thiagolocatelli/parse4j/commit/f4c024e46d55b869507f01da01a53e5d214270d1)

**Comment by raxxus85** [comment]
I'm getting "Exception in thread "main" org.json.JSONException: A JSONObject text must begin with '{' at 1 [character 2 line 1]" when attempting to signup() a user, is this related?
